src/ExpectationMaximization.js

```diff
@@ -0,0 +1,133 @@
+import {Cluster} from './Cluster';
+import Matrix from 'ml-matrix';
+import expectationMaximization from 'expectation-maximization';
+
+const defaultOptions = {
+    epsilon: 2e-16,
+    numClusters: 2,
+    seed: undefined
+};
+
+/**
+ * @class ExpectationMaximization
+ */
+export class ExpectationMaximization {
+
+    /**
+     * Constructor for Expectation Maximization
+     * @param {object} options
+     * @param {number} [options.epsilon=2e-16] : Convergence threshold for final solution.
+     * @param {number} [options.numClusters=2] : Number of clusters to find.
+     * @param {number} [options.seed=undefined] : Seed for the random generator
+     */
+    constructor(options) {
+        options = Object.assign({}, defaultOptions, options);
```

Comment:
maybe instead of using `defaultOptions` you could use object destructuring

Reply:
can you give me an example of what you want to do with `defaultOptions`

Reply:
instead of
```js
options = Object.assign({}, defalutOptions, options);
```

do something like
```
const {
  option1 = "default1",
  option2 = "default2",
  ...
} = options;
```
and specify `options = {}` in the constructor parameter so that it always defaults to an object